designsafe/static/scripts/projects/components/manage-project/manage-project.component.js

```diff
@@ -2,6 +2,7 @@ import ManageProjectTemplate from './manage-project.template.html';
 import AmendProjectTemplate from './amend-project.template.html';
 import FormOptions from './project-form-options.json';
 import FormDefaults from './project-form-defaults.json';
+import { object } from 'underscore';
```

Comment:
I believe this may have been auto imported on accident with your IDE

Reply:
removed this.